coralproject/cay
# Auth screens first pass

## What does this PR do?

I'm creating this PR so it doesn't get out of hand later. This is just the Sign In/Up forms with tests. I created the Permissions and Forgot Password components, but they're blank. I'll fill them out in the next PR.
## How do I test this PR?
- go to `/sign-in` and see a form
- go to `/sign-up` and see a form
- run `npm test`. tests should pass

@coralproject/frontend
